pandas/tests/indexes/test_numpy_compat.py

```diff
@@ -77,6 +76,11 @@ def test_numpy_ufuncs_basic(index, func):
 def test_numpy_ufuncs_other(index, func, request):
     # test ufuncs of numpy, see:
     # https://numpy.org/doc/stable/reference/ufuncs.html
+    msg = (
+        f"ufunc '{func.__name__}' not supported for the input types, and the "
+        "inputs could not be safely coerced to any supported types according "
+        "to the casting rule ''safe''"
+    )
```

Comment:
This one's also an external error from `numpy`. Even though this works, probably better to be consistent and use `tm.external_error_raised`.

Reply:
done